routing/directions_engine.hpp

```diff
@@ -18,6 +18,9 @@ class IDirectionsEngine
 public:
   virtual ~IDirectionsEngine() = default;
 
+  // @TODO(bykoianko) After fields |m_turns|, |m_times|, |m_streets| and |m_traffic|
```

Comment:
Вот так переписал:

  // @TODO(bykoianko) When fields |m_turns|, |m_times|, |m_streets| and |m_traffic|
  // are removed from class Route the method Generate() should fill
  // vector<RouteSegment> instead of corresponding arguments.